mod-source-record-manager-server/src/main/resources/vertx-default-jul-logging.properties

```diff
@@ -1,5 +1,5 @@
 handlers = java.util.logging.ConsoleHandler
 .level = ALL
 
-logger.cql2pgjson.level = ERROR
-logger.cql2pgjson.name = org.folio.cql2pgjson.CQL2PgJSON
+#logger.cql2pgjson.level = ERROR
```

Comment:
is this intentional?

Reply:
Changes in logs related to the testing process. I reverted these changes. Thank you.